index.html

```diff
@@ -0,0 +1,1402 @@
+              </div>
+              <div class="row w-100 d-inline-block mt-3 border-top py-3">
+                <div class="float-left mr-3">
+                  <a href="">
+                    <img class="img-fluid" src="./img/jon-zhou.webp" alt="Jon">
+                  </a>
+                </div>
+                <h5>
+                  <strong><a href="">The Ennui of the Civics Teachers</a></strong>
+                </h5>
+                <h6>
+                  <strong>BY FREDERICK M. HESS AND MATTHEW RICE</strong>
+                </h6>
+              </div>
+              <div class="row w-100 d-inline-block mt-3 border-top py-3">
+                <div class="float-left mr-3">
+                  <a href="">
+                    <img class="img-fluid" src="./img/john-yoo.webp" alt="John">
+                  </a>
+                </div>
+                <h5>
+                  <strong>
+                    <a href="">MLK Versus BLM in Changing Hearts and Minds</a>
+                  </strong>
+                </h5>
+                <h6>
+                  <strong>
+                    BY PAUL ROBINSON
+                  </strong>
+                </h6>
+              </div>
+              <div class="row w-100 d-inline-block mt-3 border-top py-3">
+                <div class="float-left mr-3">
+                  <a href="">
+                    <img class="img-fluid" src="./img/yaakov-peri.webp" alt="Yaakov">
+                  </a>
+                </div>
+                <a href="#">
+                  <h5>
+                    <strong>International Law, Not Terrorism, Must Be the New Paradigm</strong>
+                  </h5>
+                </a>
+                <h6 class="ml-5">
+                  <strong>BY MICHAL COTLER-WUNSH</strong>
+                </h6>
+              </div>
+            </div>
+
+
+          <!--Free Sign-Up-->
+
+
+            <div class="row card mt-4 ml-3 border-0 mx-auto d-block jumbotron p-1">
+              <div class="card-body text-center">
+                <img class="float-left m-1 mr-3" src="./img/N-icon.png" alt="N icon" width="30">
+                <h4 class="text-left">START YOUR DAY WITH OUR TOP 5 ARTICLES</h4>
+                <div class="input-group my-3 mt-5">
+                  <input class="form-control" type="text" placeholder="Email Address">
+                </div>
+                <button type="button" class="btn btn-secondary w-100 mt-2">FREE SIGN UP <i class="fas fa-angle-right"></i></button>
+              </div>
+            </div>
+        </div>
+
+        
+      </div>
+    </section>
+
+    <!--In the Magazine-->
+
+    <section class="row mx-auto d-block container-fluid">
+      <div class="row">
+        <div class="col-12 d-inline-flex justify-content-center m-sm-3 border-bottom py-3">
+          <img class="float-left m-1 mr-3" src="./img/N-icon.png" alt="N icon" width="30">
+          <h1>
+            In The Magazine
+          </h1>
+        </div>
+      </div>
```

Comment:
- [ ] The height and alignment of the images are too different. Try to make closer to the original.


| Yours | Original |
|:---:|:---:|
|![image](https://user-images.githubusercontent.com/61479648/89235387-70bd9380-d5c4-11ea-9b23-c0cd039484d4.png)| ![image](https://user-images.githubusercontent.com/61479648/89235350-5daac380-d5c4-11ea-852f-720f365fddc5.png) |